fixtures/keywords/rust/src/lib.rs

```diff
@@ -24,7 +24,7 @@ impl r#break {
 }
 
 #[allow(non_camel_case_types)]
-trait r#continue {
+pub trait r#continue {
```

Comment:
huh - if this is now necessary, is this technically a breaking change? (TBH I'm fine if it is but should maybe be documented)

Reply:
I don't think this breaks anything, the issue was at the rust level:
```
warning: private trait `continue` in public interface (error E0445)
   --> /home/bdk/uniffi-rs/ffi-converter-metadata/target/debug/build/uniffi-fixture-keywords-rust-ae8b10cd8ad52109/out/keywords.uniffi.rs:225:5
    |
225 | /     pub extern "C" fn r#keywords_rust_2de3_break_continue(
226 | |         r#ptr: *const std::os::raw::c_void,
227 | |         r#continue: uniffi::RustBuffer,
228 | |     call_status: &mut uniffi::RustCallStatus
229 | |     )  -> <std::option::Option<std::vec::Vec<Box<dyn r#continue>>> as ::uniffi::FfiConverter<crate::UniFfiTag>>::ReturnType {
    | |___________________________________________________________________________________________________________________________^
    |
    = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
    = note: for more information, see issue #34537 <https://github.com/rust-lang/rust/issues/34537>
    = note: `#[warn(private_in_public)]` on by default

warning: `uniffi-fixture-keywords-rust` (lib) generated 1 warning
```